src/Fabulous.Avalonia/Views/_StyledElement.fs

```diff
@@ -61,6 +62,20 @@ module StyledElement =
     let ActualThemeVariantChanged =
         Attributes.defineEventNoArg "StyledElement_ActualThemeVariantChanged" (fun target -> (target :?> StyledElement).ActualThemeVariantChanged)
 
+    let ThemeKey =
+        Attributes.defineSimpleScalarWithEquality<string> "StyledElement_ThemeKey" (fun _ newValueOpt node ->
+            let target = node.Target :?> StyledElement
+
+            match newValueOpt with
+            | ValueNone -> target.Theme <- null
+            | ValueSome themeKey ->
+                match Application.Current.Styles.TryGetResource(themeKey, null) with
+                | true, value ->
+                    match value with
+                    | :? ControlTheme as controlTheme -> target.Theme <- controlTheme
+                    | _ -> ()
```

Comment:
I think it might be a good idea to set the Theme to null if the theme key is not found.
This would avoid being confused by having an old theme still displayed when we expect the new one.
eg. `.themeKey("old_theme")` --> `.themeKey("new_theme")` would still display `old_theme` if `new_theme` is not found.

Reply:
Also, I recommend you print warning messages to the logger `node.TreeContext.Logger.Warn("xxx")` to help users diagnose this kind of silent errors.

Those messages are printed by the `|> Program.withLogLevel` function defined at the program level.

Reply:
Updated as requested.